app/screens/localization/CountryLanguageConfirmation.tsx

```diff
@@ -196,7 +199,7 @@ const CountryLanguageConfirmation = ({ route }: Props): any => {
       )
 
       setSponsorsData(selectedCountry);
-      const foundCountry = getCountryByCountryCode(RNLocalize.getCountry());
+      const foundCountry = allCountries.length === 1 ? allCountries?.[0] : getCountryByCountryCode(RNLocalize.getCountry());

```

Comment:
Check if allCountries is not null or undefined before accessing the length property.

`allCountries?.length`